doc/api/http2.md

```diff
@@ -30,6 +30,41 @@ can be accessed using:
 const http2 = require('http2');
 ```
 
+## Determining if crypto support is unavailable
+
+It is possible for Node.js to be built without including support for the
+`crypto` module. In such cases, attempting to `import` from `http2` or
+calling `require('http2')` will result in an error being thrown.
+
+When using CommonJS, the error thrown can be caught using try/catch:
+
+```cjs
+let http2;
+try {
+  http2 = require('http2');
+} catch (err) {
+  console.log('http2 support is disabled!');
+}
+```
+
+When using the lexical ESM `import` keyword, the error can only be
+caught if a handler for `process.on('uncaughtException')` is registered
+_before_ any attempt to load the module is made -- using, for instance,
+a preload module.
```

Comment:
Optional nit: Switch to parentheses. Using an em dash has all sorts of typographical pitfalls. For example, in this case you've put spaces around the em dash, but [Microsoft](https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/emes) and many other style guides say that an em dash should not have spaces around it. As another example, this is two dashes put together rather than an em dash character (—). This is how it was done on mechanical typewriters that didn't have an em dash character, but now we have em dash characters! 🎉  

In contrast, the conventions with parentheses are broadly understood and agreed upon.

```suggestion
_before_ any attempt to load the module is made (using, for instance,
a preload module).
```

Reply:
I've copied that from https://github.com/nodejs/node/blob/aa97c9d973b336f0e92aa3a5f60e7b24a6053c6c/doc/api/crypto.md#L55-L58, maybe we fix that first.